third-party/csp-evaluator/optimized_binary.js

```diff
@@ -0,0 +1,216 @@
+// @ts-nocheck
```

Comment:
is there any movement on optimizing this file further? And anywhere tracking that for us? I fear this could remain an 80KiB blob in our bundle indefinitely (or until someone happens to open up a bundle viewer a year or two from now).

I'm not saying `module$exports$google3$javascript$security$csp$csp_evaluator$csp.isHash=module$contents$google3$javascript$security$csp$csp_evaluator$csp_isHash` doesn't gzip/brotli well, but `Aa=f` will still usually end up smaller :)

Reply:
I was able to make the bundle smaller *and* usable by copying the source code to a new directory, compiling with tsc and bundling with js-bundle.

The problem with this is that the process isn't as simple as taking the binary from google3 and adding an export statement. There were some code modifications I had to make for it to work.

Reply:
Hey! We actually just finished publishing this to NPM so it should now be possible to add a dependency on the NPM package rather than vendoring this blob. See:

https://www.npmjs.com/package/csp_evaluator

And let me know if you run into any issues with using the package. I believe I set everything up correctly, but I'm not a JS expert. :)

Reply:
Looks like CSP Evaluator is on npm!
https://www.npmjs.com/package/csp_evaluator

We can switch to the module in a follow-up.

Reply:
Reduces devtools bundle size by 45 KB